server-src/apis.js

```diff
@@ -255,11 +255,20 @@ function initApis(config, app, db) {
   if (process.env.NODE_ENV === 'development') {
     // For easier testing, this endpoint creates a valid JWT without Oauth
     app.get('/api/devlogin', async (req, res) => {
-      logger.info('GET /devlogin: %s', JSON.stringify(req.query));
+      logger.info('GET /api/devlogin: %s', JSON.stringify(req.query));
 
       genJwtCookie(res, getUserId('dev', req.query.email));
       return res.status(200).send('success').end();
     });
```

Comment:
I've added a cleanup item to the projects list to migrate Login.js to use the new /api/oauthDevUrl and delete devlogin

Reply:
Okay, I can make that change along with these if you like…

Reply:
Done